package.json

```diff
@@ -35,7 +37,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint ."
+    "lint": "eslint src"
```

Comment:
Was there a specific file that you were trying to avoid the linter? If so, I think you can add exclusions to the `.eslintrc.json` file.

Reply:
Oh, I didn't know I could do that! I added this because it started to lint the `build/` folder when I ran npm run build, which gave me a kajillion warnings and errors... I decided to change this to 'src' because I didn't think there were any files in the root directory that need to be linted, but I think excluding the `build/` folder is a better solution. Thanks!